### ISSUE TITLE
GRANT: opt_with_grant_options

### ISSUE THREAD

**Issue by walter-weinmann** [K2InformaticsGmbH/sqlparse#39]
WITH NAME OPTION seems to make no sense with the GRANT statement:.

```
grant_def ->
 GRANT system_privilege_list opt_on_obj_clause TO grantee_commalist opt_with_grant_option
```

```
opt_with_grant_option -> '$empty'
opt_with_grant_option -> WITH GRANT OPTION
opt_with_grant_option -> WITH NAME OPTION
opt_with_grant_option -> WITH HIERARCHY OPTION
```

The Oracle SQL GRANT statement syntax diagrams can be found here:

https://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_9013.htm#SQLRF01603

**grant:**

![image](https://cloud.githubusercontent.com/assets/1956149/14274262/926f7be4-fb10-11e5-8a75-28eb4a250cab.png)

**grant_system_privilleges:**

![image](https://cloud.githubusercontent.com/assets/1956149/14274298/c0725160-fb10-11e5-9cd5-cc7d24002416.png)

**grant_object_privilleges:**

![image](https://cloud.githubusercontent.com/assets/1956149/14274310/d3eeaf36-fb10-11e5-97df-132aaa7a1ce4.png)

**Comment by walter-weinmann** [comment]
Suggested solution: delete the following grammar rule:

`with_grant_option -> WITH NAME      OPTION                                                      : strl2atom(["with", '$2', "option"]).`

**Comment by walter-weinmann** [comment]
Solved with #82.
